core/dbt/contracts/graph/manifest.py

```diff
@@ -640,6 +653,9 @@ class Manifest(MacroMethods, DataClassMessagePackMixin, dbtClassMixin):
     source_patches: MutableMapping[SourceKey, SourcePatch] = field(default_factory=dict)
     disabled: MutableMapping[str, List[GraphMemberNode]] = field(default_factory=dict)
     env_vars: MutableMapping[str, str] = field(default_factory=dict)
+    public_nodes: MutableMapping[str, PublicModel] = field(default_factory=dict)
+    project_dependencies: Optional[ProjectDependencies] = None
```

Comment:
for follow-up consideration: potentially should be moved to RuntimeConfig. Think it could be addressed as part of https://github.com/dbt-labs/dbt-core/issues/7469